modules/osdk-installing-cli.adoc

```diff
@@ -11,6 +11,151 @@ The Operator SDK has a CLI tool that assists developers in creating, building,
 and deploying a new Operator project. You can install the SDK CLI on your
 workstation so you are prepared to start authoring your own Operators.
 
+[NOTE]
+====
+This guide uses
+link:https://github.com/kubernetes/minikube#installation[minikube] v0.25.0+ as
+the local Kubernetes cluster and link:https://quay.io/[quay.io] for the public
+registry.
+====
+
+[id="osdk-installing-cli-gh-release_{context}"]
+== Installing from GitHub release
+
+You can download and install a pre-built release binary of the SDK CLI from the
+project on GitHub.
+
+.Prerequisites
+
+- link:https://docs.docker.com/install/[`docker`] v17.03+
+- OpenShift CLI (`oc`) v4.1+ installed
+- Access to a cluster based on Kubernetes v1.11.3+
+- Access to a container registry
+
+.Procedure
+
+. Set the release version variable:
++
+----
+RELEASE_VERSION=v0.7.0
+----
+
+. Download the release binary.
++
+--
+* For Linux:
++
+----
+$ curl -OJ https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+----
+
+* For macOS:
++
+----
+$ curl -OJ https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin
+----
+--
+
+. Verify the downloaded release binary.
+
+.. Download the provided ASC file.
++
+--
+* For Linux:
++
+----
+$ curl -OJ https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
+----
+
+* For macOS:
++
+----
+curl -OJ https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+----
+--
+
+.. Place the binary and corresponding ASC file into the same directory and run
+the following command to verify the binary:
++
+--
+* For Linux:
++
+----
+$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
+----
+
+* For macOS:
++
+----
+$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+----
+--
++
+If you do not have the maintainer's public key on your workstation, you will
+get the following error:
++
+----
+$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+$ gpg: assuming signed data in 'operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin'
+$ gpg: Signature made Fri Apr  5 20:03:22 2019 CEST
+$ gpg:                using RSA key <key_id> <1>
+$gpg: Can't check signature: No public key
+----
+<1> RSA key string.
++
+To download the key, run the following command, replacing `<key_id>` with the RSA
+key string provided in the output of the previous command:
++
+----
+$ gpg --recv-key "<key_id>"
```

Comment:
Got errors below when running it. @LiliC Any ideas? What the `--keyserver` value should be?
```console
mac:operator-sdk-4.1 jianzhang$ gpg --recv-key xxx
gpg: keyserver receive failed: Server indicated a failure
mac:operator-sdk-4.1 jianzhang$ ls
operator-sdk-v0.7.0-x86_64-apple-darwin     operator-sdk-v0.7.0-x86_64-apple-darwin.asc
```

Reply:
@jianzhangbjz https://github.com/operator-framework/operator-sdk/pull/1442 See this PR for the fix. cc @adellape

Reply:
@LiliC Thanks!